deepchecks/tabular/checks/model_evaluation/roc_report.py

```diff
@@ -45,7 +46,7 @@ def __init__(self, excluded_classes: List = None,
                  random_state: int = 42,
                  **kwargs):
         super().__init__(**kwargs)
-        self.excluded_classes = excluded_classes or []
+        self.excluded_classes = excluded_classes
```

Comment:
why

Reply:
so you could explicitly define to not ignore anything (currently in binary by default we ignore class 0)

Reply:
will add it also in docstring